gsa/src/web/pages/scanners/dialog.js

```diff
@@ -289,7 +289,7 @@ class ScannerDialog extends React.Component {
                       </Divider>
                     </Layout>
                     {is_edit && isDefined(state.ca_pub) && (
-                      <CertStatus info={state.ca_pub.info} />
+                      <CertStatus info={state.ca_pub_info} />
```

Comment:
I don't think this will work because the dialog state doesn't contain a ca_pub_info property. But nevertheless for me it seems that was already broken before...